readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://YriKm.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://YriKm.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="959" alt="image" src="https://github.com/user-attachments/assets/33453782-ef7c-445c-af84-6b7b0dc58ae4" />